modules/ding_gatewayf/ding_gatewayf.module

```diff
@@ -241,6 +244,31 @@ function ding_gatewayf_callback() {
 }
 
 /**
+ * Make a salt based on request parameters.
```

Comment:
This docblock is somewhat misleading. This function does not generate a [salt](https://en.wikipedia.org/wiki/Salt_(cryptography)). The salt is the value set as a variable in `settings.php`.